document-generator-accounts/src/test/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImplTest.java

```diff
@@ -91,16 +92,16 @@ void testErrorThrownWhenFailedAccountsHandler() throws HandlerException, Service
     @DisplayName("Tests the unsuccessful retrieval of document data due to error in Accounts handler")
     void testUnsuccessfulGetDocumentInfoExceptionFromAccountsHandler()
             throws HandlerException {
-        when(accountsHandlerMock.getAbridgedAccountsData(any(Transaction.class),  anyString())).thenThrow(new HandlerException("error"));
+        when(accountsHandlerMock.getAbridgedAccountsData(any(Transaction.class),  anyString(), anyString())).thenThrow(new HandlerException("error"));
 
-        assertThrows(HandlerException.class, () -> accountsHandlerMock.getAbridgedAccountsData(transaction, ""));
+        assertThrows(HandlerException.class, () -> accountsHandlerMock.getAbridgedAccountsData(transaction, "", MIME_TYPE));
```

Comment:
This is testing the mocking framework not the service

Reply:
The Mock is used to check that the HandlerException is caught when the accountsHandler is called within AccountsDocumentInfoServiceImpl. As opposed to just checking that the a DocumentInfoException is caught. The reason is to ensure that if the exception is altered it is caught within the tests.